.travis.yml

```diff
@@ -1,43 +1,71 @@
 env:
   global:
-    - RUNTESTFLAGS=-v
+    - CFLAGS="-g -O2 -fstack-protector -Wformat -Werror=format-security" LDFLAGS="-Wl,-Bsymbolic-functions -Wl,-z,relro"
+    - PGUSER="postgres"
+  matrix:
+    - POSTGRESQL_VERSION="9.5"
+    - POSTGRESQL_VERSION="10"
 
 addons:
-  postgresql: "9.6"
+  apt:
+    packages:
+      - eatmydata
 
 before_install:
+  - eval "${MATRIX_EVAL}"
+  - sudo sh -c "echo /usr/lib/libeatmydata/libeatmydata.so >> /etc/ld.so.preload"
```

Comment:
:open_mouth: :smile: 

First time I see this. I understand it should save some IO write time. Did it get noticeably better?

Reply:
It comes from upstream `svn_trunk` in travis. I basically took that one and adapted it to build for multiple PG releases, so I'm not sure :smile: